Code/GraphMol/ChemReactions/DaylightParser.cpp

```diff
@@ -87,11 +87,39 @@ std::vector<std::string> splitSmartsIntoComponents(
   }
   return res;
 }
+
+ROMol* constructMolFromString(const std::string &txt,
+    std::map<std::string,std::string> *replacements, bool useSmiles, bool sanitize) {
+  ROMol *mol;
+  if(!useSmiles){
+    mol=SmartsToMol(txt,0,false,replacements);
+    if(sanitize){
```

Comment:
If it's really intended for depiction, figuring out some sensible way to combine it with `prepareMolForDrawing()` ( https://github.com/rdkit/rdkit/blob/e79f3ad9503550fcfb9c089b9980fc1f706b7dfc/Code/GraphMol/MolDraw2D/MolDraw2DUtils.h#L36 ) might be worth considering. But that's definitely a nice-to-have.